src/System.Runtime.Extensions/src/System/Environment.Win32.cs

```diff
@@ -288,7 +288,7 @@ public static int ProcessorCount
                 // Allocate that much space
                 Debug.Assert(len > 0);
                 var buffer = new byte[len];
-                fixed (byte* bufferPtr = buffer)
+                fixed (byte* bufferPtr = &buffer[0])
```

Comment:
I suggest reverting this one.  We do have an assert above, but that's asserting an assumption about how the Win32 function behaves.